### PR TITLE
Update agent review instructions on large PRs

### PR DESCRIPTION
**Pull request by jimmygchen**
## Description

Add a "Large PR Strategy" section to `.ai/CODE_REVIEW.md`, based on the regression caught in #8842 (fix in #8843).

When `get_block_roots_from_store` changed its return type in a large PR, downstream code silently changed behavior because callers weren't re-evaluated. The new section captures general strategies for reviewing large PRs: prioritize shared type/interface changes and trace their cross-subsystem impact, and flag missing test coverage for changed behavior paths.

### CODE REVIEW

File: .ai/CODE_REVIEW.md [hunk 1]
```diff
@@ -190,6 +190,14 @@ we typically try to avoid runtime panics outside of startup."
 - Edge cases handled?
 - Context provided with errors?
 
+## Large PR Strategy
+
+Large PRs (10+ files) make it easy to miss subtle bugs in individual files. When reviewing a large PR:
+
+- **Break the review into focused areas** — group files by subsystem (e.g., networking, store, types) and review each group independently. Consider spawning separate agents for large or critical files to give them focused attention.
+- **Pay extra attention to type signature changes** — when a function's return type changes (e.g., `Vec<T>` to `Vec<(T, U)>`), trace all callers and verify that trait-dependent operations (`.unique()`, `.dedup()`, `.contains()`, `HashSet` usage) still have the intended semantics on the new type.
```
Comment (jimmygchen) [inline .ai/CODE_REVIEW.md]: I think this might be a bit over-specific and may not be useful

**Review by dapplion — APPROVED** [review]
